src/axom/quest/IntersectionShaper.hpp

```diff
@@ -948,9 +1511,10 @@ class IntersectionShaper : public Shaper
   BoundingBoxType* m_aabbs {nullptr};
   PolyhedronType* m_hexes {nullptr};
   BoundingBoxType* m_hex_bbs {nullptr};
```

Comment:
Minor (and likely out of scope of this PR):
Should we convert these arrays to `axom::Array` ?